### PR TITLE
Deploy/prepare

### PR DESCRIPTION
**Pull request by wildananugra**

### CODE REVIEW

**Review by darmawanalbert — APPROVED** [review]
Okay from my end

**Review by nuvianggaresti — APPROVED** [review]
ok

**Review by clawrencia — APPROVED** [review]

File: classifier/requirements.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ nltk
 pandas
 gensim
 python-Levenshtein
-hashlib
```
Comment (cakraocha) [inline classifier/requirements.txt:6]: I assume the hashlib deleted here because it is part of the python library right?

**Review by cakraocha — APPROVED** [review]
Just commenting the deleted hashlib in classifier/requirements.txt, but all good. Thanks!

**Pull request merged**